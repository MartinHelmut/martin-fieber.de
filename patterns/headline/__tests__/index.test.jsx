import "jest-styled-components";
import React from "react";
import renderer from "react-test-renderer";

import Headline from "../index";

describe("Headline", () => {
  test("should render", () => {
    const tree = renderer.create(<Headline>Headline text</Headline>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
