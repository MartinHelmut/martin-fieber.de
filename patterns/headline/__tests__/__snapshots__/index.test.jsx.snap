// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should render 1`] = `
<h2
  className=""
>
  Headline text
</h2>
`;
